README.md

```diff
@@ -457,6 +462,51 @@ axios.get('/user/12345', {
 })
 ```
 
+## Cancellation
+
+You can cancel a request using a *cancel token*.
+
+> The axios cancel token API is based on [cancelable promises proposal](https://github.com/tc39/proposal-cancelable-promises), which is currently at Stage 1.
+
+You can create a cancel token by passing an executor function to the `CancelToken` constructor as shown below:
+
+```js
+var Cancel = axios.Cancel;
+var CancelToken = axios.CancelToken;
+
+var cancel;
+
+axios.get('/user/12345', {
+  cancelToken: new CancelToken(function executor(c) {
+    // An executor function receives a cancel function as a parameter
+    // You can use the cancel function to cancel the request later
+    cancel = c;
+  })
+}).catch(function(thrown) {
+  if (thrown instanceof Cancel) {
```

Comment:
@rubennorte I think adding `axios.isCancel(value)` is the way to go. We can easily update it to support native `Cancel` objects (when they are available). I'll update the PR soon.